src/main/java/backend/resource/Model.java

```diff
@@ -99,7 +103,7 @@ public UpdateSignature getUpdateSignature() {
     }
 
     @Override
-    public List<TurboIssue> getIssues() {
+    public synchronized List<TurboIssue> getIssues() {
```

Comment:
And to deal with that, we synchronise the `reconcile`, `getLabels`, `setLabels` methods on the `TurboIssue` level also :-P.